plugins/hypervisors/kvm/pom.xml

```diff
@@ -45,7 +38,7 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
-      <groupId>org.libvirt</groupId>
+      <groupId>com.github.K0zka</groupId>
```

Comment:
@K0zka in that case if you'll be maintaining it then we should merge it. Since, I'm not sure I'll wait for any of the KVM people to comment on it. (Wido, Kishan, Marcus?)